doc/build.md

```diff
@@ -193,10 +193,10 @@ The optional `<content>` element defines the app's starting page in ordinary Cor
 
 In addition to the settings described in the [Cordova config.xml Guide](http://cordova.apache.org/docs/en/dev/config_ref/), Tabris.js also accepts the following custom preferences:
 
-| Name                   | Allowed Values | Default Value | Description |
-|------------------------|----------------|---------------|-------------|
-| EnableDeveloperConsole | true/false     | false         | Enables/Disables the [Tabris.js Developer Console](getting-started.md#the-developer-console).             |
-| UseStrictSSL           | true/false     | true          | Activate/Deactivate SSL certificate validation on [XHR](w3c-api.md#xmlhttprequest). When disabled self signed SSL certificates are accepted. Should be enabled in production. |
+| Name                   | Allowed Values       | Default Value | Description |
+|------------------------|----------------------|---------------|-------------|
+| EnableDeveloperConsole | true/false/$IS_DEBUG | false         | Enables/Disables the [Tabris.js Developer Console](getting-started.md#the-developer-console). Setting the value to `$IS_DEBUG` will make the value follow the value for [debug mode](#settings)|
```

Comment:
Actually, `$IS_DEBUG` is just a pre-defined variable. You could use any variable (env or specified on the [command line](https://www.npmjs.com/package/tabris-cli#--variables-replacements) that resolves to `true` or `false`. I like the idea to point to `$IS_DEBUG` here, but I'd prefer not to add it as an *allowed value*, as this could create the impression that `$IS_DEBUG` was a special *value*.